app/views/admin_merchants/index.html.erb

```diff
@@ -10,19 +10,23 @@
   <% end %>
 </div>
 
-<h2>List of All Merchants:</h2>
-<% @merchants.each do |merchant| %>
-  <div id="merchants-<%= merchant.id %>">
+<h2>All Merchants:</h2>
 
-  <h3>-<%= link_to "#{merchant.name}", "/admin/merchants/#{merchant.id}" %></h3>
+<h3>Enabled Merchants</h3>
+<% @merchants.each do |merchant| %>
+<div id="merchants-<%= merchant.id %>">
```

Comment:
This view page looks pretty good, one thing i would do is enclose the whole 'Enabled Merchants' section (header and each block included) inside of a div tag, with id="enabled-merchants". This way you can use a within block in the test, to test that only merchants with a status of 'enabled' can be found in the 'enabled merchants' section. 
The same can be done with the disabled merchants section

Reply:
There are now within block to test the enable/disable sections on the admin merchant index page.